src/api/rest/server/resource/k8scluster.go

```diff
@@ -603,6 +603,42 @@ func RestPostK8sClusterDynamic(c echo.Context) error {
 	return c.JSON(http.StatusOK, result)
 }
 
+// RestPostK8sNodeGroupDynamic godoc
+// @ID PostK8sNodeGroupDynamic
+// @Summary Create K8sNodeGroup Dynamically
+// @Description Create K8sNodeGroup Dynamically from common spec and image
+// @Tags [Kubernetes] Cluster Management
+// @Accept  json
+// @Produce  json
+// @Param nsId path string true "Namespace ID" default(default)
+// @Param k8sClusterId path string true "K8sCluster ID" default(k8scluster-01)
+// @Param k8snodegroupReq body model.TbK8sNodeGroupDynamicReq true "Request body to provision K8sNodeGroup dynamically. <br> Must include commonSpec and commonImage info. <br> (ex: {name: k8snodegroup-01, commonImage: azure+koreacentral+ubuntu22.04, commonSpec: azure+koreacentral+Standard_B2s}]}) <br> You can use /k8sclusterRecommendNode and /k8sclusterDynamicCheckRequest to get it. <br> Check the guide: https://github.com/cloud-barista/cb-tumblebug/discussions/1913"
+// @Param x-request-id header string false "Custom request ID"
+// @Success 200 {object} model.TbK8sNodeGroupInfo
+// @Failure 404 {object} model.SimpleMsg
+// @Failure 500 {object} model.SimpleMsg
+// @Router /ns/{nsId}/k8scluster/{k8sClusterId}/k8snodegroupDynamic [post]
```

Comment:
```suggestion
// @Router /ns/{nsId}/k8scluster/{k8sClusterId}/k8sNodeGroupDynamic [post]
```
need to keep api path lowerCamelCase for consistency.
(later, please check other API paths as well)

Reply:
관련 API path들을 검토하여 lowerCamelCase 형식으로 업데이트하였습니다.